### ISSUE TITLE
Roofs hiden hook rt4

### ISSUE THREAD

**Issue by ArtificialPB** [powerbot/RSBot-API#1298]
If there's a hook to determine if the roofs have been removed, could this get added? If roofs are not removed and the player is standing outside of the building some npcs and objects inside can't be clicked. I've already checked and this setting is not stored in varpbits.

**Comment by Moosheplusplus** [comment]
This will only work if "always remove roofs" isn't enabled in the runescape settings. Also, the camera will automatically remove roofs if the camera is within the building, despite if the player is in it or not.

**Comment by ArtificialPB** [comment]
This is a request for a method that determines if that game setting is selected.

**Comment by benpoulson** [comment]
@AshGreh Could we add this to the list of future hooks we'd like to add, as I agree it could be a good hook to add.

As of v184, that hook can be found at the following:
```
class: bl
field: v
static: false
descriptor: boolean
```
For now I'm closing this ticket. However, I've made a note to comment on it when this specific hook has been implemented. We have upcoming work to implement a big backlog of new and missing hooks.
